### DISCUSSION TITLE
Nous A1 device won’t turn on or do anything

### DISCUSSION THREAD

**Discussion by Dannypeja** [arendst/Tasmota#12921] [Q&A] [answered]
**issue:** Nous A1 Socket not working anymore. Doesn't connect to wifi, doesn't blink. Nothing. Like it's dead

**steps to reproduce:** flashed using tuya convert using included default firmware (not aware which exactly) but upgraded soon. Everything worked fine. I forgot reset 5 at first but then did it a couple of hours later. Plug was not under load. No power outage suffered. I may have plugged it in/ out some times.
I used the Nous A1 Template and everything worked. 

**situation right now:** when it gets power, blue and red led flash for a brief moment. After that nothing happens, as the device would be dead. No Wifi connection, thus no mqtt response. No Response when pressing button. Fast recovery power cycling doesn't work or I am doing it wrong.

**Comment by barbudor** [comment, marked as answer]
This is lacking a lot af details
Did they ever worked once? Have you been able to do initial configuration : wifi, template etc...?
Which exact firmware have you flashed? The default 9.2 available in tuya convert repository or did you found smart to use another one?

**Reply by Dannypeja** [reply]
Sorry you are right! I added info to the initial post.

**Reply by barbudor** [reply]
So it worked for some time and then suddenly it stopped working without any apparent reason ?
Do you have multiple of those ? All showing the same ?

May be the initial Tuya flashing was not fully correct.
That's all I can guess for now
I fear you have to open it up and look for a way of serial flashing.
At least if you can open and hook a USB/Serial adapter you may be able to witness the boot sequence in a serial terminal (powering from USB, not from mains!!!)

Some similar looking plugs have a hidden screw in the french earth hole if yours happen to have one.
If not, the cover is glued and may pop up with firm and slow force.

**Reply by Dannypeja** [reply]
Thanks a lot!!
I have 4, two of them show this behavior.
Tuya flash *seemed* all fine. No errors, nothing. Maybe it was the late upgrade + reset 5? 

Thanks for the tip with the hidden screw. I was wondering how to open them. I saw a similar plug where the transparent plastic has to be removed.

**Reply by Dannypeja** [reply]
I can confirm that the nous A1 has a screw in the french earth whole. Once unscrewed it comes apart easily. But then I don’t know how to get the board out. 
i think it is not needed since there are solder pads one could reach 
![CDE8CFAF-6909-419A-9AE7-04A0FC7F76D3](https://user-images.githubusercontent.com/76405661/129906211-6119ad0b-4b44-4a09-8e36-a9fdb4437b2f.jpeg)
![A29BD93C-0BD4-47D9-BDD5-F810DC8CA622](https://user-images.githubusercontent.com/76405661/129906239-fbb75cfd-d4ee-4d0c-a159-beb6f4c32c67.jpeg)

**Reply by barbudor** [reply]
This definitively looks like the Gosund SP111 I used to purchase and can't find any more. Gonna get some more. Can you tell me where you purchased them ?

See updated pinout in my next post. Shows GPIO0 On the resistor close to the edge connector

**Reply by Dannypeja** [reply]
So The GPIO0 in the image is wrong?


Fantastic. Really the best plug I had bought so far. tuya convert still works and seemingly it comes easy apart. 
I bought it off amazon germany but don't wanna share amazon urls. Just search for Nous A1. It even has tuya and tasmota in the description.

**Reply by barbudor** [reply]
GPIO0 on the ESP module is correct. What was wrong is my statement that GPIO0 was on the Button. GPIO0 is LED1
Here is an updated photo with GPIO0 on the right pin of resistor below LED1

Normally the board comes out quite easily but the antenna sticker blocks a little bit. You would need to disconnect the antenna cable and try to get it out of the way.
Getting the board out would help if you want to solder.
Otherwise, just tweak a set of header with long pins should be ok if you can keep your hand firm during the flashing.


![ESP8266 module pinout](https://user-images.githubusercontent.com/2996042/129913222-ff79f1bb-e818-4a0b-9513-dee84f034d9d.jpg)

**Reply by barbudor** [reply]
I have 8 of the Gosund SP111 branded ones. Only one showed a failure like the one you reported and needed to be serially reflashed.
Since then, all good.
Very handy and nice plug.
